src/main/java/org/elasticsearch/discovery/zen/ZenDiscovery.java

```diff
@@ -176,7 +176,7 @@ public ZenDiscovery(Settings settings, ClusterName clusterName, ThreadPool threa
         this.rejoinOnMasterGone = settings.getAsBoolean(SETTING_REJOIN_ON_MASTER_GONE, true);
 
         if (this.joinRetryAttempts < 1) {
-            throw new ElasticsearchIllegalArgumentException("'" + SETTING_JOIN_RETRY_ATTEMPTS + "' must be a positive number. got [" + this.SETTING_JOIN_RETRY_ATTEMPTS + "]");
+            throw new ElasticsearchIllegalArgumentException("'" + SETTING_JOIN_RETRY_ATTEMPTS + "' must be a positive number. got [" + SETTING_JOIN_RETRY_ATTEMPTS + "]");
```

Comment:
Shouldn't this be `this.joinRetryAttempts` in the message instead of printing the same constant twice?